File: Outros/POO UML/Aula 10/br.com.uninove.empresa/src/br/com/uninove/empresa/PessoaFisica.java
package br.com.uninove.empresa;

public class PessoaFisica extends Pessoa {

		private String rg;
		
		public PessoaFisica(String nome, String rg)
		{
			// Validações
			super(nome);
			setRG(rg);
		}
		
		public String getRG() {
			return rg;
		}

		private void setRG(String rg) {
			// Várias validações
			this.rg = rg;
		}
	
		public void mostraClasse()
		{
			System.out.println("Classe PessoaFisica");
		}
		
}

File: programacao/php/03-web/09/Exemplo/PessoaFisica.php
<?php

class PessoaFisica extends Pessoa {
    // Atributo
    private string $rg;
    
    // Comportamentos
    public function getRg() : string {
        return $this->rg;
    }

    public function setRg(string $rg) {
        $this->rg = $rg;
    }
}
?>